packages/components/src/DataList/components/DataListSearch/DataListSearch.css

```diff
@@ -4,7 +4,7 @@
 
   position: absolute;
   /* Inputs are off by 1 when put beside a button */
-  top: calc(var(--space-small) - 1px);
+  top: calc(var(--space-small) + 1px);
```

Comment:
everything looks good to me so I approved the PR. just curious though why we need this??

Reply:
oh right. should've raised that on the description. The off by 1 comment is actually miscalculated. it shouldn't be a subtraction. Otherwise, you'd get this

![image](https://github.com/GetJobber/atlantis/assets/15986172/e9bc22dc-06b0-4719-a20c-01678c8135ec)


compared to addition

![image](https://github.com/GetJobber/atlantis/assets/15986172/8e3698d4-3083-42cd-9b4a-5856d7f53bb1)